air/src/air/lagrange/mod.rs

```diff
@@ -0,0 +1,37 @@
+mod boundary;
+pub use boundary::LagrangeKernelBoundaryConstraint;
+
+mod frame;
+pub use frame::LagrangeKernelEvaluationFrame;
+
+mod transition;
+use math::FieldElement;
+pub use transition::LagrangeKernelTransitionConstraints;
+
+use crate::{AirContext, LagrangeConstraintsCompositionCoefficients};
+
+/// Represents the Lagrange kernel transition and boundary constraints.
+pub struct LagrangeKernelConstraints<E: FieldElement> {
+    pub transition: LagrangeKernelTransitionConstraints<E>,
+    pub boundary: LagrangeKernelBoundaryConstraint<E>,
+}
```

Comment:
I wonder if the Lagrange column index should be a part of this struct. Would slightly simplify the code in a few places.

Reply:
done